ecosystem/sep-0024.md

```diff
@@ -483,7 +516,8 @@ Name | Type | Description
 `operation` | string | Kind of operation (`deposit` or `withdraw`).
 `type` | string | (optional) Type of deposit or withdrawal (`SEPA`, `bank_account`, `cash`, etc...).
 `asset_code` | string | Asset code.
-`amount` | float | Amount of the asset that will be deposited/withdrawn.
+`external_asset_code` | string | The asset code of the external asset the user will send or receive, if the value of `asset_code` is `XLM`.
+`amount` | float | Amount of the asset that will be deposited or withdrawn. If `asset_code` is `XLM`, `amount` should be in units of `external_asset_code` if `operation` is `deposit` and in units of XLM if `operation` is `withdraw`.
```

Comment:
The units of `amount` should depend on the asset the user is sending to the anchor. For example, the user sends the anchor XLM for a withdraw, but a non-stellar asset for a deposit.